docs/Developers/Writing_Good_Commits.rst

```diff
@@ -98,6 +102,9 @@ Example:
     It helps not writing more functions inside ``coalib`` for this.
 
 -  Maximum of 72 chars excluding newline for *each* line.
+   The recommendation is to do this at 72 characters,
```

Comment:
I think the word 'do this' here isnt clear.
I believe you mean 'do this' = 'add a line break'.

And again, 'recommendation' is misleading.  coala has a hard requirement that `body_line_length` is 72. Longer than that isnt possible.  It will be rejected by the bots and builds.

The issue you are solving is to explain why.